### PR TITLE
Added workflow for manual deployments

### PR DESCRIPTION
**Pull request by VenkataChalla554**
## Context

Added new workflow which can be used to manual deployments

## Changes proposed in this pull request

New workflow for manual deployments.

## Guidance to review

The work will be used for review and qa this can be expanded for other environments.

## Link to Trello card

 https://trello.com/c/C1sBpxLz/1717-ittms-manual-deployment-workflow

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated or [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Comment by VenkataChalla554** [comment]
[ittms manual deployment workflow](https://trello.com/c/C1sBpxLz/1717-ittms-manual-deployment-workflow)

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    environment:
+      name: ${{ github.event.inputs.environment }}
+    concurrency: deploy_all
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: azure/login@v1
```
Comment (RMcVelia) [inline .github/workflows/deploy.yml:33]: v2

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Environment to deploy to
+        required: true
+        type: choice
+        default: review
+        options:
+        - qa
+        - review
```
Comment (RMcVelia) [inline .github/workflows/deploy.yml:13]: can you add all envs

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Fetch secrets from key vault
+        if: ${{ github.event.inputs.environment != 'review' }}
+        uses: azure/CLI@v1
```
Comment (RMcVelia) [inline .github/workflows/deploy.yml:39]: v2

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (VenkataChalla554) [reply .github/workflows/deploy.yml:13]: Added all environments

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (VenkataChalla554) [reply .github/workflows/deploy.yml]: Fixed

File: .github/workflows/deploy.yml (hunk 3 above)
Comment (VenkataChalla554) [reply .github/workflows/deploy.yml]: fixed

**Review by RMcVelia — APPROVED** [review]

**Pull request merged**